compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -25,30 +25,49 @@
 namespace
 {
 
+/// @brief Returns the number of BroadcastTo(Custom) node among cop's inputs
```

Comment:
This PR comes from the following discussion.
- https://github.com/Samsung/ONE/pull/12509#discussion_r1470611294